stories/Button/Guideline.mdx

```diff
@@ -0,0 +1,169 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  Primary,
+  Controls,
+  Unstyled,
+  Source,
+} from '@storybook/blocks';
+import { Button } from '../../src/lib/components/buttonv2/Buttonv2.component';
+
+import * as ButtonStories from './button.stories';
+
+<Meta of={ButtonStories} />
+
+# Button
+
+Buttons are used to trigger an action or event when clicked.
+
+## Size & style
+
+### Style
+
+- Height: 2rem
+- Border-radius: 3px
+- Minimum width: 5rem
+
+### Padding
+
+Button’s label measurement:
+
+- Line height: 1.25rem
+- Vertical padding: 0.5rem
+- Horizontal padding of 1rem
+- Space between icon & label: 0.5rem
+
+### Button spacing
+
+- Horizontal spacing: 1rem
+- Vertical spacing: 0.75rem
+- Space between 2 buttons: 1.5rem - when there is no space constraint
+
+## Usage
```

Comment:
I'd remove the do/don't, as we are kind of mixing them.
Here is the proposal:


Button labels should be as short and clear as possible and should describe the action the button performs.

- Use one or two words if possible,
- Remove most prepositions and articles (a, an, the).
- Examples: Cancel, Close, Create, Delete, Edit, Learn more, Review, Save, Study, View scores, etc.,
- Stick to using verbs (Complete, Start, Finish, Search) or a simple verb + noun combination for buttons (Next page, Submit post, Learn more),
- Maintain labelling method consistency across all of your buttons,
- Don't use punctuation or exclamation marks (!),
- And, most importantly, don't write "Click here".



About the sentence case: we are not really using this type of case for buttons, as most often, we capitalize the noun after the verb (Create Folder, Create Node, Edit Location). So we are closer to "Title Case" for the button labels.

Reply:
Should I add the title case in the list ? something like : - Capitalize the noun after the verb (Create Folder, Create Node, Edit Location).

Reply:
yes, good idea!